deepforest/main.py

```diff
@@ -416,6 +416,7 @@ def predict_file(self, csv_file, root_dir, savedir=None, color=None, thickness=1
 
     def predict_tile(self,
                      raster_path=None,
+                     path= None,
```

Comment:
Remove the extra space after the `=`

Reply:
We can also add the `path` as the last parameter to minimize the signature changes. But I am okey with leaving it in that position.

Reply:
Good to consider this @henrykironde, but I think that since  `raster_path` and `image` should never both be specified there was no way for folks to not use named arguments after `raster_path`. Therefore there shouldn't be any backwards breaking changes and I think leaving `path` first makes the most sense. Does that sound right?

Reply:
Correct, that skipped my mind.